readme.md

```diff
@@ -715,6 +715,7 @@
 - [WP-CLI](https://github.com/schlessera/awesome-wp-cli#readme) - Command-line interface for WordPress.
 - [Credit Modeling](https://github.com/mourarthur/awesome-credit-modeling#readme) - Methods for classifying credit applicants into risk classes.
 - [Ansible](https://github.com/KeyboardInterrupt/awesome-ansible#readme) - A Python-based, open source IT configuration management and automation platform.
+- [Translations](https://github.com/mbiesiad/awesome-translations#readme) - Adaptation of a text or a speech from one language to many others.
```

Comment:
I think you could come up with a better description on what "translation" is.

---

> one language to many others.

It doesn't have to be "many others". It could be one other.

Reply:
I wrote more options and variants
- "Transfer of the meaning of a text from one language to another."
- "The transfer of the meaning of a text from one language to another."
- "The process of translating words or text from one language into another." - From Oxford English Dictionary
- "Translation is the communication of the meaning of a source-language text by means of an equivalent target-language text." - Wikipedia
- "The communication of the meaning of a source-language text by means of an equivalent target-language text."
- "Communication of the meaning of a source-language text by means of an equivalent target-language text."

Tell me if there is one you like.

Reply:
I would go with:

> The transfer of the meaning of a text from one language to another.